src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -129,74 +132,137 @@ export default class BarGraph extends React.Component<IBarGraphProps, {colors: s
-                        }
+                            const colors = tooltipModel.labelColors[i];
+                            let style = 'background-color: ' + tooltipModel.labelColors[0].backgroundColor;
+                            style += '; border-color:' + colors.borderColor + '; border-width: 1px';
+                            // style += '; border-width: 1px';
+                            var span = '<span class="chartjs-tooltip-key" style=' + style + '></span>';
+                            innerHtml += '<tr><td>' + span + body + ' cases</td></tr>';
+                        });
+                        innerHtml += '</tbody>';
+
+                        const tableRoot = tooltipEl.querySelector('table');
+                        tableRoot!.innerHTML = innerHtml;
                     }
+
+                    // `this` will be the overall tooltip
+                    const position = this._chart.canvas.getBoundingClientRect();
+
+                    // Display, position, and set styles for font
+                    tooltipEl.style.opacity = '1';
+                    tooltipEl.style.left = position.left + tooltipModel.caretX - 10 + 'px';
+                    tooltipEl.style.top = position.top + tooltipModel.caretY + 5 + 'px';
+                    tooltipEl.style.padding = tooltipModel.yPadding + 'px ' + tooltipModel.xPadding + 'px';
                 }
+
+            },
+            scales: {
+                xAxes: [{ stacked: true }],
+                yAxes: [{
+                    stacked: true,
+                    gridLines: {display: false},
+                    ticks: {fontSize: 11}
+                }]
             },
+            legend: { display: false }
+        };
 
-            series: _.flattenDeep(cancerTypeStudiesArray.map((cancerStudySet, i:number) => (
-                cancerStudySet.studies.map((cancerStudy: CancerStudy, j: number) => {
-                    const {name, studyId, allSampleCount} = cancerStudy;
-                    const max = cancerStudySet.studies.length;
-                    const colors = ['1E36BF', '128C47', 'BF2231', '7D1FBF', 'BF7D15'];
-                    const color = this.lightenDarkenColor(colors[i%5], (max-j)/max * 120);
-                    return {
-                        borderColor: '#F1F6FE',
-                        name,
-                        studyId,
-                        data: i === 0 ? [{y: allSampleCount, color}] : [...Array(i).fill(0),{y: allSampleCount, color}]
-                    };
-                })
-            )))
+        const canvas:any = document.getElementById("chart");
 
+        new Chart(canvas.getContext("2d"), {
+            type: 'horizontalBar',
+            data,
+            options
         });
+
     }
 
     render() {
-        return <div ref={el => this.chartTarget = el} />;
+        return (
+            <div id="canvas-holder" style={{position: 'relative'}}>
```

Comment:
always avoid using ids.  eventually it will come back to haunt you when more than one instance needs to be created.  to select DOM nodes inside the scope of a component instance, use React refs.